spec/support/matchers/have_setting.rb

```diff
@@ -0,0 +1,21 @@
+require 'rspec/expectations'
+
+RSpec::Matchers.define :have_setting do |expected|
+  match do |actual|
+    getter = actual.respond_to? expected
+
+    setter_method = expected.to_s + "="
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.